index.html

```diff
@@ -1,48 +1,297 @@
+                    <div class="item5"><h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                    <div class=" item5 btn-group">
+                        <button class="languages"> Ruby on rails</button>
+                        <button class="languages"> css</button>
+                        <button class="languages"> JavaScript</button>
+                        <button class="languages"> html</button>
+                    </div>
+
+                    
+                    <div  class="item5"><button class="see-project">See project</button></div>
+                    
+                
+
+
+
+                </section>
+        </div>
+
+        <div>
+            <section id="works4" class="container6">
+
+
+                <div class="item6"> <img alt="work4" src="img/work4.jpg"></div>
+
+                    
+                                
+                <div  class="item6"><h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                <div class=" item6 btn-group">
+                    <button class="languages"> Ruby on rails</button>
+                    <button class="languages"> css</button>
+                    <button class="languages"> JavaScript</button>
+                    <button class="languages"> html</button>
+                </div>
+
+                
+                <div class="item6"><button class="see-project">See project</button></div>
+            
+            
+
+
+
+            </section>
+        </div>
+
+        <div>
+            <section id="works5" class="container7">
+
+
+                <div class="item7"> <img alt="work5" src="img/work5.png"> </div>
+
+                    
+                                
+                <div class="item7"> <h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                <div class=" item7 btn-group">
+                    <button class="languages"> Ruby on rails</button>
+                    <button class="languages"> css</button>
+                    <button class="languages"> JavaScript</button>
+                    <button class="languages"> html</button>
+                </div>
+
+                
+                <div class="item7"><button class="see-project">See project</button></div>
+            
+            
+
+
+
+            </section>
+        </div>
+
+        <div>
+            <section id="works6" class="container8">
+
+
+                <div class="item8"> <img alt="work6" src="img/work6.jpg"> </div>
+
+                    
+                                
+                <div class="item8"><h2>Multi-Post Stories<br> Gain+Glory</h2></div>
+
+
+                    <div class=" item8 btn-group">
+                        <button class="languages"> Ruby on rails</button>
+                        <button class="languages"> css</button>
+                        <button class="languages"> JavaScript</button>
+                        <button class="languages"> html</button>
+                    </div>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li, let us wrap the technologies with the `ul` and `li`